src/pydrex/core.py

```diff
@@ -49,12 +49,45 @@ class MineralPhase(IntEnum):
 
 @unique
 class DeformationRegime(IntEnum):
-    """Deformation mechanism regimes."""
+    r"""Ordinals to track distinct regimes of dominant deformation mechanisms.
+
+    The mechanism of deformation that dominates accommodation of plastic deformation
+    depends in general both on material properties such as grain size and mineral phase
+    content as well as on thermodynamic properties such as temperature, pressure and
+    water content (via its fugacity).
```

Comment:
I should reword it, I guess if anything the "water content" (ppm? number of substituted H and O? are these equivalent?) is a proxy for the fugacity and not the other way around, right?

Reply:
Water content is a general term to designate a measure of water in a system. It could be a concentration, a fugacity (which is a partial pressure if I am not mistaken), an atomic or molecular measure, etc...